controllers/database_helper.go

```diff
@@ -276,24 +275,32 @@ func generateTemplatedSecrets(dbcr *kciv1beta1.Database, databaseCred database.C
 
 	logrus.Infof("DB: namespace=%s, name=%s creating secrets from templates", dbcr.Namespace, dbcr.Name)
 	for key, value := range templates {
```

Comment:
(minor)

think sometimes it's also useful for longer code in the for loop, to give better names to the vars. like `secretKey` `secretTemplate`. 

But only minor, no need to fix.

Reply:
I'll just fix it in the next pull request :)